tcf_website/tests/test_review.py

```diff
@@ -126,3 +129,84 @@ def test_unauthorized_user_delete(self):
         # trying to delete a review you don't have access to should result in
         # 403 - forbidden
         self.assertEqual(response.status_code, 403)
+
+
+class ModelReviewTests(TestCase):
+    """Tests for the Review model."""
+
+    def setUp(self):
+        setup(self)
+
+    def test_count_votes(self):
+        """Test for count votes method"""
+
+        expected_sum = 1
+        self.assertEqual(self.review1.count_votes(), expected_sum)
+
+    def test_count_votes_no_votes(self):
+        """Test for count votes method for when there are no votes"""
+
+        expected_sum = 0
+        self.assertEqual(self.review2.count_votes(), expected_sum)
+
+    def test_upvote(self):
+        """Test for upvote method verify with count_votes"""
+
+        self.review1.upvote(self.user4)
+        expected_sum = 2
+        self.assertEqual(self.review1.count_votes(), expected_sum)
+
+    def test_upvote_already_upvoted(self):
+        """Test for upvote method verify with count_votes"""
```

Comment:
This docstring is the same as the one for the method above.

Reply:
Good catch! As you can tell most of the time I was copy and pasting and changing them to fit the current test cause I got lazy